index.js

```diff
@@ -44,6 +46,13 @@ function JSONdb(filePath, options) {
   } else {
     this.filePath = filePath;
   }
+  
+  // Option validating
+  if (options) {
+    if (options.stringify && options.jsonSpaces) {
+      throw new Error('jsonSpaces cannot be used with custom stringifier. Try using a wrapper instead.');
+    }
+  }
```

Comment:
There's no need to error out and break the user's flow, it's fine to silently ignore `jsonSpaces` when a custom stringifier is used.